src/main/scala/com/exasol/cloudetl/kafka/consumer/KafkaRecordConsumer.scala

```diff
@@ -57,11 +58,64 @@ class KafkaRecordConsumer(
         )
       } while (shouldContinue(recordOffset, recordCount, totalRecordCount))
     } catch {
+      case exception: IllegalStateException =>
+        throw new KafkaConnectorException(
+          ExaError
+            .messageBuilder("E-KCE-20")
+            .message("Error consuming Kafka topic {{TOPIC}} data. ", topic)
+            .message("Consumer is not subscribed to any topic or assigned any partition.")
```

Comment:
Maybe "assigned to"? Or rephrase in some other way.

Reply:
I would like to keep `subscribe` since it is the term used in Kafka. (`subscribe to a topic`, `get assigned topic partition`)

But I changed it to make more clearer.